lib/mods_display/fields/copyright_date.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module ModsDisplay
+  class CopyrightDate < Field
+    def fields
+      date_fields(:copyrightDate)
```

Comment:
DRYed these up by having the date_fields method in field.rb

Reply:
Can you put these in the `DateField` superclass instead of `Field`?